serving/src/main/java/feast/serving/store/bigquery/model/FeatureSetInfo.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2018-2019 The Feast Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package feast.serving.store.bigquery.model;
+
+import java.util.List;
+
+public class FeatureSetInfo {
```

Comment:
Is it possible for us to perhaps rename this class to something that is a little bit more descriptive? `FeatureSetInfo` doesn't really express intent.

Perhaps if you could add comments here to describe all the functionality that it should provide, then I can also help to suggest some names.

Reply:
`FeatureSetInfo` contains information about the feature set to be passed to the query templater to construct the query. I'm happy to take any suggestions for names if `FeatureSetInfo` doesn't convey enough information

Reply:
`FeatureSetInfo` seems mostly okay to me. To describe it from the other way around, how about `QueryParameters`?

Reply:
My issue with `QueryParameters` is that `QueryParameters` implies it is the full set of query template parameters, but this struct contains only the params specific to the feature set, and there are other query params outside of what's stored inside the object.